package.json

```diff
@@ -36,6 +36,9 @@
     "ts-standard": "^10.0.0",
     "typescript": "^4.3.5"
   },
+  "peerDependencies": {
+    "pino-pretty": "^5.1.2"
+  },
```

Comment:
(nit) cool, so adding it as a peer dependency here doesn't remove it from the devDependencies? As in having it here just tells npm that if it is used, then download it, if not don't download it? Or do we need to remove it from devDependencies too?

Reply:
It should've been removed from dev dependencies. I've fixed that.

Reply:
Just kidding. The tests needed it, so I added it back in. Anything in devDependencies isn't included when bundles. But since it's needed in the application using the library rather than the library itself, it should be a peerDependency.